### PR TITLE
Feature/nexteuropa 6150

### PR DESCRIPTION
**Pull request by gervasek**

### CODE REVIEW

File: profiles/common/modules/features/splash_screen/splash_screen.info [hunk 1]
```diff
@@ -2,7 +2,7 @@ name = Splash screen
 description = Provide a homepage to choose language and store language in cookie.
 core = 7.x
 package = Multisite Features
-version = 7.x-2.0
+version = 7.x-2.02
 project = splash_screen
 features[features_api][] = api:2
 features[user_permission][] = administer splash screen
```
Comment (senseca) [inline profiles/common/modules/features/splash_screen/splash_screen.info]: Should we not add the variable 'splash_screen_homepage_hijacking'?
features[variable][] = splash_screen_homepage_hijacking

File: tests/features/splash_screen.feature [hunk 2]
```diff
@@ -44,9 +44,26 @@ Feature: Splash Screen features
...
+    Then I should see the success message "The configuration options have been saved."
+    When I go to "/"
+    Then I should see an "body.not-front.page-splash" element
+    When I go to "admin/config/regional/splash_screen_settings"
+    And I uncheck the box "edit-splash-screen-homepage-hijacking"
+    And I press the "Save" button
+    Then I should see the success message "The configuration options have been saved."
+    When I go to "/"
+    Then I should see an "body.front" element
+    And I should see the heading "Welcome to NextEuropa"
```
Comment (senseca) [inline tests/features/splash_screen.feature]: Behat tests doesn't work.
See continuousphp.
In my dev environment, I have the following message : 
"  [Behat\Gherkin\Exception\ParserException]
  Expected Feature, but got text: "Feature: Splash Screen features" in file: /ec/dev/server/fpfis/webroot/sources/senseca/platform-dev/tests/features/splash_screen.feature"

**Comment by senseca** [comment]
The feature 'splash screen' should be activated automatically (needed by language switcher).

**Comment by degliwe** [comment]
Closed in favor of https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-6322

**Pull request closed without merging**